src/jquery.i18n.js

```diff
@@ -143,7 +143,7 @@
 				locale = $.i18n().locale;
 			}
 			if ( typeof source === 'string' &&
-				source.split( '.' ).pop() !== 'json'
+				source.split( '?' )[0].split( '.' ).pop() !== 'json'
```

Comment:
Looks good. Some comments:
1. This would be a nice time to add a comment there why we do that split
2. The CI is failing because as per the style guide, you need space after [ and space before ] in [0]

Reply:
Sorry, I'm not familiar enough good with Github, so I don't know what I can do for including my code in master branch. Create new pull request where these whitespaces added?

Reply:
Usually it is adding a new commit and pushing it to same branch for this pull request.

I will do the fix for this PR myself